### PR TITLE
Add connector total count next to add from connector button

### PR DESCRIPTION
**Pull request by AdityaKhatri**
- Depends on [1169](https://github.com/the-deep/server/pull/1169)

## Changes

* Add total sources count next to button

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/views/Tagging/index.tsx [hunk 1]
```diff
@@ -104,7 +140,7 @@ function Tagging(props: Props) {
                             onClick={showUnifiedConnectorModal}
                             name={undefined}
                         >
-                            Add sources from connectors
+                            {`Add sources from connectors (${newSourcesCount ?? '0'} new)`}
```
Comment (tnagorra) [inline app/views/Tagging/index.tsx]:
```suggestion
                            {`Add sources from connectors (${newSourcesCount ?? 0} new)`}
```

File: app/views/Tagging/index.tsx [hunk 2]
```diff
@@ -38,6 +58,20 @@ function Tagging(props: Props) {
...
+
+    const {
+        data,
+    } = useQuery<ConnectorSourcesCountQuery, ConnectorSourcesCountQueryVariables>(
+        CONNECTOR_SOURCES_COUNT,
+        {
+            skip: !variables,
+            variables,
+        },
+    );
```
Comment (tnagorra) [inline app/views/Tagging/index.tsx:73]: Don't we need to refresh this after some sources are added from connectors?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/views/Tagging/index.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply app/views/Tagging/index.tsx:73]: I think its fine.

**Review by subinasr — APPROVED** [review]

File: app/views/Tagging/index.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply app/views/Tagging/index.tsx:73]: i do already

**Review by tnagorra — APPROVED** [review]

**Pull request merged**